airbyte-webapp/src/packages/cloud/services/thirdParty/fullstory/useFullStory.tsx

```diff
@@ -1,11 +1,13 @@
+import type { User } from "packages/cloud/lib/domain/users";
```

Comment:
Currently, we try to follow a pattern where libs imports are going first. Could you please move it after the absolute one? 

I understand that it doesn't bring much value and instead, we should try to set up prettier for such types of things, but I believe that for now we should keep it as everywhere in the code.